### ISSUE TITLE
[FEATURE] Add ability to add a dashboard filter by clicking a TSVB dataset

### ISSUE THREAD

**Issue by tender-barbarian** [opensearch-project/OpenSearch-Dashboards#5590]
**Is your feature request related to a problem? Please describe.**

TSVB graph does not allow dashboard filter to be created by clicking on a value.

**Describe the solution you'd like**

Clicking on a value in TSVB graph should add new dashboard filter.

**Additional context**

https://github.com/elastic/kibana/issues/11291
https://github.com/elastic/kibana/issues/83892

This feature has been alread added in Kibana.

**Comment by paveljanda** [comment]
Agree. It's really useful. Without it, the TSVB panel is half-unusable for drilldowns.
